core/kits/documents/tabs/site-identity.php

```diff
@@ -21,6 +21,7 @@ public function get_title() {
 
 	public function register_tab_controls() {
 		$custom_logo_id = get_theme_mod( 'custom_logo' );
+		$custom_logo_src = wp_get_attachment_image_src( $custom_logo_id, 'full' );
```

Comment:
Is there a situation that the id exists, but the src doesn't?

Reply:
Yes, a delete attachment.

Reply:
In this case, you shouldn't print the id at all

Reply:
In this case, do you need to print the ID?

Reply:
yes, in order the media control will show that a value is exist.

Reply:
There are still some conflicts that need to be resolved

Reply:
Done